pkg/plugin/run.go

```diff
@@ -55,7 +63,11 @@ func (e *AutoscaleEnforcer) startPermitHandler(ctx context.Context) error {
 			"[autoscale-enforcer] Received autoscaler-agent request (client = %s) %+v",
 			r.RemoteAddr, req,
 		)
-		resp, statusCode, err := e.handleAgentRequest(req)
+		nodeName := "<none>"
+		resp, statusCode, err := e.handleAgentRequest(req, &nodeName)
+		finalStatus = statusCode
+		e.metrics.validResourceRequests.WithLabelValues(nodeName, req.ProtoVersion.String(), strconv.FormatBool(req.Metrics != nil)).Inc()
```

Comment:
That's fair. I don't know enough to know how concerned we should be about that, but node names seems like it would be really useful for figuring out what's going wrong.

Reply:
I think I'd lean toward pulling the protocol version (it'll be disvoverable in a pinch,) and it seems like most of the time it'd be fine (but also useless) *except* if we had to deploy a few times to fix bugs, (e.g. in some kind of fire-fighting situation) where we'd run the risk of DoS'ing the metrics db, just when we need it the most?

Reply:
Makes sense, done.